### PR TITLE
Feat: Notes development

### PR DESCRIPTION
**Pull request by abhijeetekad**
Updated notes making template,
New features:
- archive note, unarchive note
- delete note =>restore deleted note.

chrome-capture (16)

### CODE REVIEW

File: src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -33,13 +33,21 @@
...
+
+.sideBar-items {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  margin-left: 1.5rem;
+  padding-left: 0.5rem;
+}
+.sideBar-items:hover {
+  background-color: rgb(116, 118, 143);
```
Comment (shivanigangadharan) [inline src/Components/Sidebar/Sidebar.css:51]: suggestion to use color variables throughout the project to maintain consistency, can create a common styles.css file

File: src/App.js [hunk 2]
```diff
@@ -5,19 +5,20 @@ import { NewNote } from "./Components/NewNote/NewNote";
...
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<LandingPage />} />
+        <Route path="/mockman" element={<MockmanEs />} />
         <Route path="/main-page" element={<MainPage page={<NewNote />} />} />
-        <Route path="pin-note" element={<MainPage page={<PinNote />} />} />
+        <Route path="archive" element={<MainPage page={<Archive />} />} />
 
         <Route path="/all-notes" element={<MainPage page={<AllNotes />} />} />
```
Comment (shivanigangadharan) [inline src/App.js:23]: i like the use of page attribute inside the element attribute.

File: src/Components/NewNote/NewNote.jsx [hunk 3]
```diff
@@ -1,74 +1,157 @@
...
-            id: "",
-          });
-          dispatchNoteData({ type: "NEW_NOTE", payload: noteInfo });
-        }}
-      >
-        <div className="form-catageory">
-          <div className="form-details">
-            <label>Note Title : </label>
+    <>
+      {/* <button onClick={() => setExtended(true)}> add note</button> */}
```
Comment (shivanigangadharan) [inline src/Components/NewNote/NewNote.jsx]: can remove comment if not reqd

**Review by shivanigangadharan — APPROVED** [review]
Just 1 suggestion that you can improve your folder structure later on for eg All notes or Archive notes should come under the pages folder and not component folder as these are having individual routes. 
Overall the code looks great, the functionalities are also working properly in the deploy preview, can merge this.

File: src/Components/Sidebar/Sidebar.css (hunk 1 above)
Comment (abhijeetekad) [reply src/Components/Sidebar/Sidebar.css:51]: yes

File: src/Components/NewNote/NewNote.jsx (hunk 3 above)
Comment (abhijeetekad) [reply src/Components/NewNote/NewNote.jsx]: yes.. done

File: src/App.js (hunk 2 above)
Comment (abhijeetekad) [reply src/App.js:23]: Thank you for the suggestion. Will keep in mind while refactoring the code.

File: src/Components/AllNotes/AllNotes.jsx [hunk 4]
```diff
@@ -1,37 +1,51 @@
...
 import "./AllNotes.css";
 import DeleteIcon from "@mui/icons-material/Delete";
 import { useNote } from "../../Context/NoteContext";
-
+import PushPinIcon from "@mui/icons-material/PushPin";
+import NoNotesImg from "../../Assets/NoNotes.png";
 function AllNotes() {
   const { stateNoteData, dispatchNoteData } = useNote();
-
+  console.log(stateNoteData.note.length);
```
Comment (Manpreet-hub) [inline src/Components/AllNotes/AllNotes.jsx]: can remove console.

**Review by Manpreet-hub — APPROVED** [review]
Everything looks good and well structure

File: src/Components/AllNotes/AllNotes.jsx (hunk 4 above)
Comment (abhijeetekad) [reply src/Components/AllNotes/AllNotes.jsx]: yes

**Pull request merged**